frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -17,12 +16,24 @@ import { columns as cols } from './columns';
 import { Table } from 'components/Table';
 import service from '../service';
 import { FaFolderOpen, FaFolder } from 'react-icons/fa';
-import { Properties } from 'features/projects/list/properties';
+import { Buildings } from './buildings';
 import { useRouterFilter } from 'hooks/useRouterFilter';
+import { FaFileExcel, FaFileAlt } from 'react-icons/fa';
+import styled from 'styled-components';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+import { ReactComponent as BuildingSvg } from 'assets/images/icon-business.svg';
+import { ReactComponent as LandSvg } from 'assets/images/icon-lot.svg';
+import { PropertyTypes } from './FilterBar';
 
 const getPropertyReportUrl = (filter: IPropertyFilter) =>
   `${ENVIRONMENT.apiUrl}/reports/properties?${filter ? queryString.stringify(filter) : ''}`;
 
+const FileIcon = styled(Button)`
+  background-color: #fff !important;
```

Comment:
Any reason for `important`

Reply:
grrr, yes because without it, another `!important` will not allow me to override...

Reply:
Hopefully when the button story gets done we'll have a set of variants to use and we won't need to keep overriding the defaults for the primary variant all over the place.